Error running migrations Docker won't start

<!--
Please fill out the following template to make solving your problem easier and faster for us.
This is only a guideline. If you think that parts are unnecessary for your issue, feel free to remove them.

Remember to hide/redact personal or confidential information,
such as passwords, IP addresses, and DNS names as appropriate.
-->

### Subject of the issue
On pulling the latest testing image I can't get it to start it shows the following in the logs

[23-10-2023 09:19:26.270][panic][ERROR] thread 'main' panicked at 'Error running migrations: QueryError(DieselMigrationName { name: "2023-10-21-221242_add_cipher_key", version: MigrationVersion("20231021221242") }, DatabaseError(Unknown, "You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '\"key\" TEXT' at line 2"))': src/db/mod.rs:471

   0: vaultwarden::init_logging::{{closure}}

   1: std::panicking::rust_panic_with_hook

   2: std::panicking::begin_panic_handler::{{closure}}

   3: std::sys_common::backtrace::__rust_end_short_backtrace

   4: rust_begin_unwind

   5: core::panicking::panic_fmt

   6: core::result::unwrap_failed

   7: vaultwarden::db::mysql_migrations::run_migrations

   8: vaultwarden::db::DbPool::from_config

   9: vaultwarden::main::{{closure}}

  10: rocket::async_main

  11: vaultwarden::main

  12: std::sys_common::backtrace::__rust_begin_short_backtrace

  13: std::rt::lang_start::{{closure}}

  14: std::rt::lang_start_internal

  15: main

  16: <unknown>

  17: __libc_start_main

  18: _start

### Deployment environment

Synology DS918+     =========================================================================================
    Preferably, use the `Generate Support String` button on the admin page's Diagnostics tab.
    That will auto-generate most of the info requested in this section.
    =========================================================================================
-->

<!-- The version number, obtained from the logs (at startup) or the admin diagnostics page -->
<!-- This is NOT the version number shown on the web vault, which is versioned separately from vaultwarden -->
<!-- Remember to check if your issue exists on the latest version first! -->
* vaultwarden version: Testing build as of 1027 on 23rd October 2023. Unable to start to get version.

<!-- How the server was installed: Docker image, OS package, built from source, etc. -->
* Install method: Docker on Synology NAS

* MySQL/MariaDB or PostgreSQL version: MariaDB latest

* Other relevant details:

### Steps to reproduce
<!-- Tell us how to reproduce this issue. What parameters did you set (differently from the defaults)
and how did you start vaultwarden? -->

Pulled latest testing image and issue happens. 

### Expected behaviour
<!-- Tell us what you expected to happen -->
Docker container starts up
### Actual behaviour
<!-- Tell us what actually happened -->

### Troubleshooting data
<!-- Share any log files, screenshots, or other relevant troubleshooting data -->
Logs are above

--- a/migrations/mysql/2023-10-21-221242_add_cipher_key/up.sql
+++ b/migrations/mysql/2023-10-21-221242_add_cipher_key/up.sql
@@ -1,2 +1,2 @@
-ALTER TABLE ciphers 
-ADD COLUMN "key" TEXT;
+ALTER TABLE ciphers
+ADD COLUMN `key` TEXT;

